source/Mlos.Python/mlos/Optimizers/ExperimentDesigner/UtilityFunctions/SeriesDifferenceUtilityFunction.py

```diff
@@ -0,0 +1,98 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+import numpy as np
+import pandas as pd
+from scipy.stats import t
+from mlos.Logger import create_logger
+from mlos.Optimizers.OptimizationProblem import SeriesObjective
+from mlos.Optimizers.ExperimentDesigner.UtilityFunctions.UtilityFunction import UtilityFunction
+from mlos.Optimizers.RegressionModels.MultiObjectiveRegressionModel import MultiObjectiveRegressionModel
+from mlos.Optimizers.RegressionModels.Prediction import Prediction
+from mlos.Spaces import SimpleHypergrid, ContinuousDimension, CategoricalDimension, Point
+from mlos.Spaces.Configs.ComponentConfigStore import ComponentConfigStore
+from mlos.Tracer import trace
+
+
+series_difference_utility_function_config_store = ComponentConfigStore(
+    parameter_space=SimpleHypergrid(
+        name="series_difference_utility_function_config",
+        dimensions=[
+            # I might use some of these from ConfidenceBoundUtilityFunction... They seem pretty reasonable.
+            CategoricalDimension(name="utility_function_name", values=["lower_confidence_bound_on_improvement", "upper_confidence_bound_on_improvement"]),
+            ContinuousDimension(name="alpha", min=0.01, max=0.5)
+        ]
+    ),
+    default=Point(
+        utility_function_name="upper_confidence_bound_on_improvement",
+        alpha=0.01
+    )
+)
+
+
+class SeriesDifferenceUtilityFunction(UtilityFunction):

```

Comment:
Could you please add a comment here describing how this works and what's the intent?